distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -0,0 +1,35 @@
+:title: Ingest Command
+:type: dataManagement
+:status: published
+:summary: Ingesting data from the ${command-console}.
+:parent: Ingesting Data
+:order: 00
+
+== {title}
+
+The ${command-console} has a command-line option for ingesting files.
+
+[NOTE]
+====
+Ingesting with the console ingest command will only create and store a metacard and not the resource itself.
```

Comment:
"Ingesting with the console ingest command creates a metacard in the catalog, but does not copy the resource to the content store."

Reply:
fixed.